syndeo-template/src/main/java/com/google/cloud/syndeo/common/ProviderUtil.java

```diff
@@ -117,11 +117,19 @@ public static PCollectionRowTuple applyConfigs(
       Collection<TransformSpec> specs, PCollectionRowTuple tuple) {
     for (TransformSpec spec : specs) {
       SchemaTransform transform = spec.provider.from(spec.configuration);
+
+      if (tuple.getAll().containsKey("errors")) {
+        // TODO(pabloem): Deal with errors.
+      }
       // We know we only deal with transforms with either 0 or 1 input so we know how to connect the
       // collections. To sanity check we should confirm the output collections match expected.
-      if (tuple.getAll().size() == 1) {
+      if (tuple.getAll().size() == 1 || tuple.getAll().containsKey("output")) {
```

Comment:
nit: should we define "errors" and "output" as constant strings (static) ?

Reply:
done thanks!